source/core/replica-set-high-availability.txt

```diff
@@ -15,29 +19,244 @@ Replica Set High Availability
    :depth: 1
    :class: singlecol
 
-:term:`Replica sets <replica set>` provide high availability using
-automatic :term:`failover`. Failover allows a :term:`secondary` member
-to become :term:`primary` if the current primary becomes unavailable.
+:term:`Replica sets <replica set>` use elections to select the 
+:term:`primary` for the cluster. Replica set elections support
+high availability by allowing clusters to automatically recover 
```

Comment:
I'd use "resume" and not "recover".  The app can recover from the failover, the cluster can resume operations.

Reply:
Or you could say the cluster recovers from a fault in a node, the network, etc. and then resumes normal operations.  But recovering normal operations implies that it will do something to make sure that any normal operations that were underway will still succeed; it won't.  That's up to the app and its retry logic and/ or write queuing etc. to handle.